apps/examples/testcase/ta_tc/st_things/stc/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# For a description of the syntax of this configuration file,
```

Comment:
Any reason for naming the folder 'stc'? 
In other sub-modules, 'itc' and 'utc' are there. I assume they represent integration and unit test cases.
Could you please check whether we can keep our test cases with the same folder names?

Reply:
Since we are writing tests for testing functional, regression and unit tests, named the folder as system test cases(stc). I think its better.